sdk/textanalytics/azure-ai-textanalytics/tests/testcase.py

```diff
@@ -17,6 +17,8 @@
 from devtools_testutils.cognitiveservices_testcase import CognitiveServicesAccountPreparer
 from azure_devtools.scenario_tests import ReplayableTest
 
+REGION = 'westcentralus'
```

Comment:
should that be read from an environment variable instead? feels weird to hard code it.

Reply:
I think it's ok to hard code it, since it's just for our tests. If I make it an environemnt variable, I'd have to add it as a secret for the live pipeline runs and I really don't think it's worth it